examples/openid-connect/app.js

```diff
@@ -0,0 +1,133 @@
+const Koa = require("koa");
```

Comment:
Currently, we have only Koa app example. I don't think we should have both (because there is no significant difference) but would like to know others' thoughts.

Reply:
When you say "both", what do you mean?

Reply:
I mean having two examples - koa and express.

Reply:
Cool, thanks for the clarification. I will resolve this comment.